en/docs/guides/access-delegation/oauth-pushed-authorization-requests.md

```diff
@@ -0,0 +1,77 @@
+# Pushed Authorization Requests (PAR)
+
+Pushed Authorization Request (PAR) is an OAuth 2.0 specification that secures the way of initiating an authorization flow.
+
+The PAR specification defines the `/par` endpoint in the authorization server, and enables an OAuth client to push the payload of an authorization request directly to this endpoint. The endpoint in response, will return a one-time use `request_uri` which acts as a reference to the registered authorization request.
+
+When making the authorization request to the authorization endpoint, it is sufficient to send the `request_uri` instead of sending the complete authorization payload thus protecting the payload from being exposed to the user agent via query parameters.
```

Comment:
need not to be payload IMO. POST authorization only deals with payload and generally authorize requests are GET and hence not sure we can refer as payload?

Reply:
We need to update this statement. Its not only for `the payload from being exposed to the user agent via query parameters` lets check the details from the specification. or even removing this statement completely is fine. However, we need to have the information in the 1st part of the statement on sending the `request_uri`

Reply:
> need not to be payload IMO. POST authorization only deals with payload and generally authorize requests are GET and hence not sure we can refer as payload?

PAR spec seems to use the term `payload`, so should be fine.